src/model/CollieConfig.ts

```diff
@@ -0,0 +1,49 @@
+import { Logger } from "../cli/Logger.ts";
+
+export class CollieConfig {
+  static CONFIG_FILE_PATH = ".collie/config.json";
```

Comment:
f: resolve config file path relative to CollieRepository dir. 
Here we are adding more dependencies on CWD

Reply:
I made CollieRepository a dependency